app/uk/gov/hmrc/eusubsidycompliancestub/models/undertakingResponses/AmendUndertakignApiResponse.scala

```diff
@@ -0,0 +1,47 @@
+/*
```

Comment:
Name of this file should be `AmendUndertakingApiResponse`

Reply:
🤦‍♀️ yeah 👍